internal/actions/check.go

```diff
@@ -49,37 +45,23 @@ func cleanupExcessWatchtowers(containers []container.Container, client container
 	for _, c := range allContainersExceptLast {
 		if err := client.StopContainer(c, 10*time.Minute); err != nil {
 			// logging the original here as we're just returning a count
-			logrus.Error(err)
+			logrus.WithError(err).Error("Could not stop a previous watchtower instance.")
 			stopErrors++
 			continue
 		}
 
 		if cleanup {
 			if err := client.RemoveImageByID(c.ImageID()); err != nil {
-				// logging the original here as we're just returning a count
-				logrus.Error(err)
-				cleanupErrors++
+				logrus.WithError(err).Warning("Could not cleanup watchtower, possibly because of other watchtowers instances in other scopes.")
```

Comment:
```suggestion
				logrus.WithError(err).Warning("Could not cleanup watchtower images, possibly because of other watchtowers instances in other scopes.")
```